src/mailchimp/mailchimp_manager.py

```diff
@@ -25,12 +30,13 @@ class MailchimpManager:
         TODO: detail public attributes
     """
 
-    def __init__(self):
+    def __init__(self, api_key, server):
         """
         Constructor
         """
-        raise NotImplementedError
-    
+   	self.server = server
+	self.api_key = api_key
```

Comment:
These attributes should be protected using an underscore. See https://towardsdatascience.com/private-protected-attributes-in-python-demystified-once-and-for-all-9456d4e56414